src/NUnitFramework/framework/Internal/Execution/CommandBuilder.cs

```diff
@@ -57,6 +57,12 @@ public static TestCommand MakeOneTimeSetUpCommand(TestSuite suite, List<SetUpTea
                 if (changes.Length > 0)
                     command = new ApplyChangesToContextCommand(command, changes);
             }
+            if (suite.Method!=null)
+            {
+                IApplyToContext[] changes = (IApplyToContext[])suite.Method.GetCustomAttributes(typeof(IApplyToContext), true);
```

Comment:
You don't need to change this, but for future reference, we prefer `var` when the type is obvious as it is in this case. On a side not, we really should create a generic extension method for `GetCustomAttributes<T>`